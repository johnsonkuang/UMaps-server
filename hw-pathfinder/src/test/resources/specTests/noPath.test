# tests no path found
CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A D 10.0
AddEdge simpleGraph A C 1.0
AddEdge simpleGraph C D 1.0
AddEdge simpleGraph D C 1.0

FindPath simpleGraph A B