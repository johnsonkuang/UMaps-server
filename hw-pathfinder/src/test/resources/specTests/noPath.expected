# tests no path found
created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 10.000 from A to D in simpleGraph
added edge 1.000 from A to C in simpleGraph
added edge 1.000 from C to D in simpleGraph
added edge 1.000 from D to C in simpleGraph

path from A to B:
path not found